pkg/build/android.go

```diff
@@ -41,14 +57,16 @@ func (a android) build(params Params) (ImageDetails, error) {
 		return details, fmt.Errorf("sysctl file is not supported for android images")
 	}
 
+	buildCfg, err := parseConfig(params.Build)
+	if err != nil {
+		return details, fmt.Errorf("error parsing android configs: %v", err)
```

Comment:
Nit: we don't really make any use of it in the project, but in general it's nicer to nest errors with `%w`, so that error wrapping-unwrapping begins to work out oft the box.

Reply:
Makes sense! Fixed.